config/locales/server.en.yml

```diff
@@ -2348,9 +2348,9 @@ en:
     secure_uploads_max_email_embed_image_size_kb: "The size cutoff for secure images that will be embedded in emails if the 'secure uploads allow embed in emails' setting is enabled. Without that setting enabled, this setting has no effect."
     slug_generation_method: "Choose a slug generation method. 'encoded' will generate percent encoding string. 'none' will disable slug at all."
 
-    enable_emoji: "Enable emoji"
+    enable_emoji: "Toggle the display and use of emojis in your Discourse instance. When enabled, users have the ability to insert emojis into their posts and other text-related fields, enhancing expressive communication. This also enables the emoji autocomplete function in the text editor. If disabled, emojis will not be rendered and users will not be able to access or use them."
```

Comment:
This is so very wordy... I'm not sure it's better than the old, concise "Enable emoji" label.

Reply:
We are trying to get rid of the redundant "enable_x" -> "Enable X" descriptions. It is a bit wordy though, will cut it down :+1: